tests/test_Config.py

```diff
@@ -205,14 +214,16 @@ def testValidate(self):
     def testRangeFieldConstructor(self):
         """Test RangeField constructor's checking of min, max"""
         val = 3
-        self.assertRaises(ValueError, pexConfig.RangeField, "", int, default=val, min=val, max=val - 1)
-        self.assertRaises(ValueError, pexConfig.RangeField, "", float, default=val, min=val, max=val - 1e-15)
+        self.assertRaises(ValueError, pexConfig.RangeField, "doc", int, default=val, min=val, max=val - 1)
+        self.assertRaises(
+            ValueError, pexConfig.RangeField, "doc", float, default=val, min=val, max=val - 1e-15
+        )
```

Comment:
If I have to be picky, I'd say this should be in its own commit because these two lines would pass regardless of your main change.

Reply:
What would be a good commit message for it?

Reply:
"Add missing docstrings for configs in unit tests" may be? It'd make sense to include just the new code changes in `python/lsst/pex/config/config.py` and the new unit test you added to be one commit ("Require non-empty docstring for config parameters"), and everything else in "Add missing docstrings for configs in unit tests".

Reply:
I implemented the current best practice for this assert using separate `with` blocks in my `testDocstring()` method but left the pre-existing occurrences untouched since there were too many. They probably need their own ticket.

Reply:
> "Add missing docstrings for configs in unit tests" may be? It'd make sense to include just the new code changes in `python/lsst/pex/config/config.py` and the new unit test you added to be one commit ("Require non-empty docstring for config parameters"), and everything else in "Add missing docstrings for configs in unit tests".

Done!